### PR TITLE
Deploy scripts phase 2

### PR DESCRIPTION
**Pull request by Thomas-Smets**

### CODE REVIEW

**Review by acollette — DISMISSED** [review]
lgtm

File: script/Constants/DeployConstants.sol [hunk 1]
```diff
@@ -80,88 +80,145 @@ library DeployBytes {
...
+    uint16 public constant wbtc_collFact_eth = 7600;
+    uint16 public constant wbtc_collFact_usdc = 8600;
+    uint16 public constant wbtc_liqFact_eth = 8400;
+    uint16 public constant wbtc_liqFact_usdc = 9400;
+    uint112 public constant wbtc_exposure_eth = 0;
+    uint112 public constant wbtc_exposure_usdc = 0;
+
+    uint16 public constant cbeth_collFact_eth = 9100;
+    uint16 public constant cbeth_collFact_usdc = 8100;
+    uint16 public constant cbeth_liqFact_eth = 9500;
```
Comment (j-vp) [inline script/Constants/DeployConstants.sol:127]: cbeth_eth has a higher liq factor than eth_eth?

File: script/Constants/DeployConstants.sol [hunk 2]
```diff
@@ -80,88 +80,145 @@ library DeployBytes {
...
+    uint16 public constant cbeth_liqFact_usdc = 9400;
```
Comment (j-vp) [inline script/Constants/DeployConstants.sol:128]: 94 seems very high for cbeth_usdc? compare with eth_usdc which is 85

File: script/Constants/DeployConstants.sol [hunk 3]
```diff
@@ -80,88 +80,145 @@ library DeployBytes {
...
+    uint16 public constant stg_collFact_eth = 6000;
+    uint16 public constant stg_collFact_usdc = 5500;
+    uint16 public constant stg_liqFact_eth = 7200;
+    uint16 public constant stg_liqFact_usdc = 7000;
+    uint112 public constant stg_exposure_eth = 1; // Cannot be deposited as primary asset, but still as yield source
+    uint112 public constant stg_exposure_usdc = 1; // Cannot be deposited as primary asset, but still as yield source
+
+    uint16 public constant uniswapV3AM_riskFact_eth = 9800;
+    uint16 public constant uniswapV3AM_riskFact_usdc = 9800;
+    uint112 public constant uniswapV3AM_exposure_eth = uint112(1_000_000 * 1e18);
```
Comment (j-vp) [inline script/Constants/DeployConstants.sol:204]: 1m may be on the low side

File: script/TransferOwnership.s.sol [hunk 4]
```diff
@@ -7,35 +7,63 @@ pragma solidity 0.8.22;
...
         erc20PrimaryAM = ERC20PrimaryAM(ArcadiaContractAddresses.erc20PrimaryAM);
         chainlinkOM = ChainlinkOM(ArcadiaContractAddresses.chainlinkOM);
-        liquidator = ILiquidator(ArcadiaContractAddresses.liquidator);
+        uniswapV3AM = UniswapV3AM(ArcadiaContractAddresses.uniswapV3AM);
+        stargateAM = StargateAM(ArcadiaContractAddresses.stargateAM);
+        stakedStargateAM = StakedStargateAM(ArcadiaContractAddresses.stakedStargateAM);
     }
 
     function run() public {
         uint256 ownerPrivateKey = vm.envUint("OWNER_PRIVATE_KEY");
```
Comment (j-vp) [inline script/TransferOwnership.s.sol:38]:
```suggestion
        uint256 ownerPrivateKey = vm.envUint("PRIVATE_KEY_DEPLOYER_BASE");
```

File: script/Constants/DeployConstants.sol (hunk 1 above)
Comment (Thomas-Smets) [reply script/Constants/DeployConstants.sol:127]: @zekiblue

File: script/Constants/DeployConstants.sol (hunk 2 above)
Comment (Thomas-Smets) [reply script/Constants/DeployConstants.sol:128]: @zekiblue

File: script/Constants/DeployConstants.sol (hunk 3 above)
Comment (Thomas-Smets) [reply script/Constants/DeployConstants.sol:204]: @zekiblue

**Review by acollette — APPROVED** [review]

**Pull request merged**